### ISSUE TITLE
Help HMGU with submission of samples

### ISSUE THREAD

**Issue by ESapenaVentura** [ebi-ait/dcp-ingest-central#1013]
**Description of the issue**

To help HMGU complete their submission of reads/metagenomes and start the process to send the data through the MGnify pipelines, I would like to help HMGU submit their samples to BioSamples.

**Action items**
- [x] Upload the sent spreadsheet to the [MICROBE AIT folder](https://docs.google.com/spreadsheets/d/15ST1nvwpJb5_7iS2ACGKBmY6Ve_iJsH-/edit?usp=drive_link&ouid=105945877951113459382&rtpof=true&sd=true)
- [x] Review the spreadsheet.
   - [ ] If there are substantial changes needed, correct, send back and wait for review.
- [x] Upload their samples to BioSamples
- [x] Send back spreadsheet with accessions
- [x] Open ticket to start conversation around ENA discussion

**Comment by ESapenaVentura** [comment]
Reviewed the spreadsheet and the values, and overall it looks fine!

No "substantial" changes needed - In order to accelerate their submission, I am going to upload the metadata to BSD and send them back a spreadsheet with the accessions. However, over the review, there are a couple of questions we need to tackle:
- They have a lot of QC information about the reads (%GC, %Q20, etc). Need to investigate if we store that information in BSD or if we should store it in ENA (I think ENA generates this info automatically when processing reads - not sure how important it is for us to keep it. For @Jeena-Rajan and myself to discuss)
- These are not cryopreserved samples, so all the cryopreservation fields can't be filled. For now, we have filled it with the keyword "not provided", but this needs to be ammended.
- Timepoints  are in a similar fashion - need to be corrected. But, since these seem to not be cryopreserved - What is the timepoint?
- Forgot to send them the "elevation" and "depth" fields with the spreadsheet, so I am going to ask her if she could fill it out

In summary, we should probably investigate these samples further, and if we can't find more info about them, set-up a short meeting with Pamela to clarify questions.

As an addendum, these are the things I changed in the spreadsheet:
- Moved site to “geographic location (region and locality)
- Used “sample_id” as “name”
- Deleted column “target” (Not filled)
- Deleted column “LUI_2023” (Not filled)

**Comment by ESapenaVentura** [comment]
I have shared the spreadsheet with the accessions with Pamela.

There's still a couple of things we need to discuss, so I have suggested in the email to have a meeting next week
